src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -293,9 +290,10 @@ fieldset.schemaform-field-template {
 }
 
 .help-phone-number-link {
-    font-weight: bold;
-    text-decoration: none;
-    margin: 0.5em 0;
+  font-weight: bold;
+  text-decoration: none;
+  margin: 0.5em 0;
+  word-break: keep-all;
```

Comment:
This prevents the phone number from wrapping across two lines in the modal. This behavior is approved by @emilywaggoner.